src/react/WindowVirtualizer.tsx

```diff
@@ -92,6 +109,10 @@ export interface WindowVirtualizerProps {
    * @defaultValue "div"
    */
   item?: keyof JSX.IntrinsicElements | CustomItemComponent;
+  /**
+   * If you put an element before virtualizer, you have to define its height with this prop.
+   */
+  startMargin: number;
```

Comment:
This prop is not required by WindowVirtualizer, because it is automatically calculated from DOM.

```suggestion
```

Reply:
It seemed like this was necessary, as in, without it my scrolls were off, but I might have been confused. will check.

Reply:
great, looks like I was confused.